### DISCUSSION TITLE
CSS isn't working can anyone help please

### DISCUSSION THREAD

**Discussion by Chloefrith** [community/community#107554] [Other Feature Feedback, Questions, & Ideas] [answered]
Edit: I have solved the problem thanks to Shawnarun! Thanks to whoever else answered just a small mistake that needed to be sorted by someone who could actually spot it 😂

**Comment by Shawnarun** [comment, marked as answer]
Hi Rename your Css file to  style.css and 
[change](url) your Css path to this   "css/style.css" in Html file
![Capture](https://github.com/community/community/assets/79635674/e3350a25-183f-4f54-a677-6f72fe4afae9)

**Reply by Chloefrith** [reply]
Thank you so much!! I just did this and it’s finally worked 🤦

**Comment by naveenalla3000** [comment]
ok, make sure that you add index.css/styles.css file to your index.html file with link tag at head section.

after checking the repo i have observed that bad naming convention to css file in "css/css" it shoud be ''css/css.css"(you have to add .css to Cascading Style Sheets)
